WordPress/Jetpack/Classes/Utility/DataMigrator.swift

```diff
@@ -20,7 +20,7 @@ enum DataMigrationError: LocalizedError {
 
     var errorDescription: String? {
         switch self {
-        case .databaseCopyError: return "The database couldn't be copied from shared directory"
+        case .databaseCopyError: return "The database couldn't be copied to/from shared directory"
```

Comment:
cc: @salimbraksa 🙂

Reply:
@dvdchr Yes, thanks for including this tweak in this PR 🙇